src/core/agent/services/signifyNotificationService.ts

```diff
@@ -240,35 +243,38 @@ class SignifyNotificationService extends AgentService {
       opType: OperationPendingRecordType;
     }) => void
   ) {
+    this.pendingOperations = await this.operationPendingStorage.getAll();
     // eslint-disable-next-line no-constant-condition
     while (true) {
-      const pendingOperations = await this.operationPendingStorage.getAll();
-      if (pendingOperations.length > 0) {
-        for (const pendingOperation of pendingOperations) {
+      if (this.pendingOperations.length > 0) {
```

Comment:
Ahh, I forgot something: we need to factor in KERIA going offline. Can you copy the format of this from `onNotificationStateChanged`?